crates/holochain_p2p/src/types/actor.rs

```diff
@@ -7,101 +7,71 @@ use holochain_types::prelude::ValidationReceiptBundle;
 use kitsune2_api::{DhtArc, SpaceId, StoredOp};
 use std::collections::HashMap;
 
-/// Get options help control how the get is processed at various levels.
-/// Fields tagged with `[Network]` are network-level controls.
-/// Fields tagged with `[Remote]` are controls that will be forwarded to the
-/// remote agent processing this `Get` request.
+/// Get options used to control how data fetching over the network is performed.
 #[derive(Clone, Debug)]
-pub struct GetOptions {
-    /// `[Network]`
-    /// How many remote nodes should we make requests of / aggregate.
-    /// Set to `None` for a default "best-effort".
-    pub remote_agent_count: Option<u8>,
-
-    /// `[Network]`
-    /// Timeout to await responses for aggregation.
-    /// Set to `None` for a default "best-effort".
-    /// Note - if all requests time-out you will receive an empty result,
-    /// not a timeout error.
+pub struct NetworkRequestOptions {
+    /// Make requests to this number of remote agents in parallel.
+    ///
+    /// When `GetOptions::as_race` is `true`, the first response received will be returned.
+    /// When `GetOptions::as_race` is `false`, responses will be aggregated until the timeout is
+    /// reached. This is not implemented.
+    ///
+    /// Defaults to `3`.
+    pub remote_agent_count: u8,
+
+    /// Timeout within which responses must arrive.
+    ///
+    /// When `None` is specified, the conductor settings will be used to determine the timeout.
     pub timeout_ms: Option<u64>,
 
-    /// `[Network]`
-    /// We are interested in speed. If `true` and we have any results
-    /// when `race_timeout_ms` is expired, those results will be returned.
-    /// After `race_timeout_ms` and before `timeout_ms` the first result
-    /// received will be returned.
+    /// Whether to treat the get as a race, returning the first response received.
+    ///
+    /// Defaults to `true`.
     pub as_race: bool,
-
-    /// `[Network]`
-    /// See `as_race` for details.
-    /// Set to `None` for a default "best-effort" race.
-    pub race_timeout_ms: Option<u64>,
 }
 
-impl Default for GetOptions {
+impl Default for NetworkRequestOptions {
     fn default() -> Self {
         Self {
-            remote_agent_count: None,
+            remote_agent_count: 3,
             timeout_ms: None,
             as_race: true,
-            race_timeout_ms: None,
         }
     }
 }
 
-impl GetOptions {
+impl NetworkRequestOptions {
     /// Using defaults is dangerous in a must_get as it can undermine determinism.
     /// We want refactors to explicitly consider this.
     pub fn must_get_options() -> Self {
         Self {
-            remote_agent_count: None,
+            remote_agent_count: 1,
```

Comment:
This is the key change, along with where it's applied in `holochain_p2p::spawn::actor::get_random_peers_for_location`